src/Tribe/Featured_Events.php

```diff
@@ -67,4 +67,26 @@ public function is_featured_query( WP_Query $query = null ) {
 
 		return (bool) $query->get( 'featured' );
 	}
+
+	/**
+	 * Indicates if 'featured' is set to a positive value either in the URL query
+	 * or the posted data (if any).
+	 *
+	 * @return bool
+	 */
+	public function featured_events_requested() {
+		$featured_request_var = tribe_get_request_var( 'featured', false );
+
+		switch ( strtolower( $featured_request_var ) ) {
+			case 'true':
```

Comment:
I added a [helper here](https://github.com/moderntribe/tribe-common/pull/288) as we don't seem to have an existing standalone helper for this that I could find.